app/config/config.yml.dist

```diff
@@ -234,7 +234,8 @@ htmlcleaner:
 #     autoconfirm: false
 
 # Define the file types (extensions to be exact) that are acceptable for upload
-# in either 'file' fields or through the 'files' screen.
+# in either 'file' fields or through the 'files' screen. Note that certain file-
+# types like `.php` are never acceptable, even if they are in this list.
```

Comment:
Specify exactly which are not accepted, "like php" is not enough.

Reply:
Updated. :-)